src/game/objects/har.c

```diff
@@ -2119,9 +2103,8 @@ int har_act(object *obj, int act_type) {
         return 0;
     }
 
-    int oldstate = h->state;
-
-    af_move *move = match_move(obj, prefix, h->inputs);
+    char truncated_inputs[2] = {h->inputs[0], '\0'};
```

Comment:
Maybe a clarifying comment here? I think what this is doing is keeping the input buffer un-truncated as long as you have input a new key in the last 9 ticks? So as long as you keep the input buffer "hot" with fresh input, it won't be considered truncated?

Reply:
Yeah, that's what it's doing. A comment is in order.

If you don't have any suggestions, I'll write one in the morning.

Reply:
I'll add one if you're otherwise occupied.

Reply:
Thanks for adding the comment, it reads well.